worker-configuration.d.ts

```diff
@@ -1,3 +1,6 @@
 interface Env {
   ANTHROPIC_API_KEY: string;
+  OPENAI_API_KEY: string;
+  GROQ_API_KEY: string;
```

Comment:
One last thing to approve:
add GEMINI_API_KEY here and also on .env.example

Reply:
thank you！
I responded with  [9e5e73a](https://github.com/stackblitz/bolt.new/pull/733/commits/9e5e73a0096b50aae64703d8823ec97aec00dea9)